content/en/docs/tasks/security/authz-tcp/index.md

```diff
@@ -105,85 +134,42 @@ Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpa
 * The **Book Reviews** section on the lower right of the page includes an error message **"Ratings service is
   currently unavailable"**.
 
-This is because Istio authorization is "deny by default", which means that you need to explicitly
-define access control policies to grant access to the MongoDB service.
+## Enforce workload-level access control for TCP traffic
 
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
-
-## Enforcing access control on TCP service
-
-Now let's set up service-level access control using Istio authorization to allow `v2` of `ratings`
-to access the MongoDB service.
-
-Run the following command to apply the authorization policy:
+The command creates a `bookinfo-ratings-v2` policy that grants the access to the MongoDB workload
+at port 27017 for `cluster.local/ns/default/sa/bookinfo-ratings-v2` service account
+that represents the `ratings-v2` workload:
 
 {{< text bash >}}
-$ kubectl apply -f @samples/bookinfo/platform/kube/rbac/mongodb-policy.yaml@
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: bookinfo-ratings-v2
+spec:
+  selector:
+    matchLabels:
+      app: mongodb
+  rules:
+  - from:
+    - source:
+        principals: ["cluster.local/ns/default/sa/bookinfo-ratings-v2"]
+    to:
+    - operation:
+        ports: ["27017"]
+EOF
 {{< /text >}}
 
-Once applied, the policy has the following effects:
-
-* Creates the following `mongodb-viewer` service role, which allows access to the MongoDB service on port 27017.
-
-    {{< text yaml >}}
-    apiVersion: "rbac.istio.io/v1alpha1"
-    kind: ServiceRole
-    metadata:
-      name: mongodb-viewer
-      namespace: default
-    spec:
-      rules:
-      - services: ["mongodb.default.svc.cluster.local"]
-        constraints:
-        - key: "destination.port"
-          values: ["27017"]
-    {{< /text >}}
-
-* Creates the following `bind-mongodb-viewer` service role binding, which assigns the `mongodb-viewer` role
-to the `bookinfo-ratings-v2` service.
-
-    {{< text yaml >}}
-    apiVersion: "rbac.istio.io/v1alpha1"
-    kind: ServiceRoleBinding
-    metadata:
-      name: bind-mongodb-viewer
-      namespace: default
-    spec:
-      subjects:
-      - user: "cluster.local/ns/default/sa/bookinfo-ratings-v2"
-      roleRef:
-        kind: ServiceRole
-        name: "mongodb-viewer"
-    {{< /text >}}
-
 Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). You should see the following sections:
 
 * **Book Details** on the lower left side, which includes: book type, number of pages, publisher, etc.
 * **Book Reviews** on the lower right side, which includes: red stars.
 
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
-
 ## Cleanup
 
 *   Remove Istio authorization policy configuration:
```

Comment:
This is not the only step needed to return the deployment to the state before the task. Please point readers to the instructions to remove a Bookinfo deployment too.